extensions/CMakeLists.txt

```diff
@@ -92,6 +92,7 @@ set(ACE_VERSION_MAJOR 3)
 set(ACE_VERSION_MINOR 12)
 set(ACE_VERSION_REVISION 3)
 EXECUTE_PROCESS(COMMAND git rev-parse --verify HEAD
+    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
Btw this didn't work with CMake integrated into visual studio, because the build and workdir would be in a temp directory somewhere else and won't be a git dir.
Forcing work directory to be in source directory fixes that.